src/frontend/frontend.ts

```diff
@@ -151,6 +158,11 @@ class App {
       this.searchDisabled = false;
     }
   }
+
+  themeChanged(newTheme) {
+    this.theme = newTheme;
+    // TODO: do stuff here to persist the theme selection
```

Comment:
@sumitarora let's do this in a separate PR. @clbond can you open an issue for the storing the value of this setting?